libr/bin/bin.c

```diff
@@ -997,9 +1002,9 @@ static void r_bin_file_free(void /*RBinFile*/ *bf_) {
 		a->sdb_addrinfo = NULL;
 	}
 	free (a->file);
+	a->o = NULL;
 	r_list_free (a->objs);
 	r_list_free (a->xtr_data);
-	r_bin_object_free (a->o);
 	memset (a, 0, sizeof (RBinFile));
```

Comment:
I can do that, but it is needed to zero the memory used on the heap, removing it may bring up some other bugs, are you sure about that?